### ISSUE TITLE
Missing dependency org.glassfish.jaxb:txw2:2.4.0-b180608.0325

### ISSUE THREAD

**Issue by lislei** [javaee/metro-jax-ws#1253]
When upgrading jax-ws from 2.3.0.2 to version 2.3.1 there is a issue with the transitive dependency to txw2. 
It appears maven central is missing the `org.glassfish.jaxb:txw2:2.4.0-b180608.0325` artefact.

Here is what my `gradle dependencies` report:
```
jaxwsTools
\--- com.sun.xml.ws:wscompile:2.3.1
     +--- org.glassfish.jaxb:jaxb-jxc:2.3.1
     |    +--- org.glassfish.jaxb:jaxb-xjc:2.3.1
     |    |    +--- org.glassfish.jaxb:jaxb-runtime:2.3.1
     |    |    |    +--- javax.xml.bind:jaxb-api:2.3.1
     |    |    |    |    \--- javax.activation:javax.activation-api:1.2.0
     |    |    |    +--- org.glassfish.jaxb:txw2:2.3.1 -> 2.4.0-b180608.0325 FAILED
     |    |    |    +--- com.sun.istack:istack-commons-runtime:3.0.7
     |    |    |    +--- org.jvnet.staxex:stax-ex:1.8
     |    |    |    |    \--- javax.activation:activation:1.1
     |    |    |    +--- com.sun.xml.fastinfoset:FastInfoset:1.2.15
     |    |    |    \--- javax.activation:javax.activation-api:1.2.0
     |    |    +--- org.glassfish.jaxb:xsom:2.3.1
     |    |    |    \--- com.sun.xml.bind.external:relaxng-datatype:2.3.1
     |    |    +--- org.glassfish.jaxb:codemodel:2.3.1
     |    |    +--- com.sun.xml.bind.external:rngom:2.3.1
     |    |    |    \--- com.sun.xml.bind.external:relaxng-datatype:2.3.1
     |    |    +--- com.sun.xml.dtd-parser:dtd-parser:1.4
     |    |    +--- com.sun.istack:istack-commons-tools:3.0.7
     |    |    |    \--- org.apache.ant:ant:1.10.2
     |    |    |         \--- org.apache.ant:ant-launcher:1.10.2
     |    |    +--- com.sun.istack:istack-commons-runtime:3.0.7
     |    |    \--- com.sun.xml.bind.external:relaxng-datatype:2.3.1
     |    +--- org.glassfish.jaxb:jaxb-runtime:2.3.1 (*)
     |    +--- com.sun.istack:istack-commons-runtime:3.0.7
     |    \--- org.glassfish.jaxb:txw2:2.3.1 -> 2.4.0-b180608.0325 FAILED
     +--- org.glassfish.jaxb:jaxb-xjc:2.3.1 (*)
     +--- com.sun.xml.ws:rt:2.3.1
     |    +--- javax.annotation:javax.annotation-api:1.3.2
     |    +--- javax.activation:javax.activation-api:1.2.0
     |    +--- com.sun.xml.ws:policy:2.7.5
     |    |    +--- org.glassfish.jaxb:txw2:2.4.0-b180608.0325 FAILED
     |    |    +--- javax.annotation:javax.annotation-api:1.3.2
     |    |    \--- com.sun.activation:javax.activation:1.2.0
     |    +--- org.glassfish.jaxb:jaxb-runtime:2.3.1 (*)
     |    +--- org.glassfish.gmbal:gmbal-api-only:3.1.0-b001
     |    |    \--- org.glassfish.external:management-api:3.0.0-b012
     |    +--- org.jvnet.staxex:stax-ex:1.8 (*)
     |    +--- com.sun.xml.stream.buffer:streambuffer:1.5.6
     |    |    \--- org.jvnet.staxex:stax-ex:1.8 (*)
     |    +--- org.jvnet.mimepull:mimepull:1.9.10
     |    +--- com.fasterxml.woodstox:woodstox-core:5.1.0
     |    |    \--- org.codehaus.woodstox:stax2-api:4.1
     |    +--- org.codehaus.woodstox:stax2-api:4.1
     |    +--- org.glassfish.ha:ha-api:3.1.9
     |    +--- com.sun.xml.messaging.saaj:saaj-impl:1.5.0
     |    |    +--- javax.xml.soap:javax.xml.soap-api:1.4.0
     |    |    +--- org.jvnet.mimepull:mimepull:1.9.8 -> 1.9.10
     |    |    \--- org.jvnet.staxex:stax-ex:1.8 (*)
     |    +--- javax.xml.ws:jaxws-api:2.3.1
     |    |    +--- javax.xml.bind:jaxb-api:2.3.1 (*)
     |    |    +--- javax.xml.soap:javax.xml.soap-api:1.4.0
     |    |    \--- javax.annotation:javax.annotation-api:1.3.2
     |    +--- javax.xml.bind:jaxb-api:2.3.1 (*)
     |    +--- javax.xml.soap:javax.xml.soap-api:1.4.0
     |    \--- javax.jws:javax.jws-api:1.1
     +--- com.sun.istack:istack-commons-runtime:3.0.6 -> 3.0.7
     +--- com.sun.xml.ws:policy:2.7.5 (*)
     +--- javax.xml.ws:jaxws-api:2.3.1 (*)
     +--- javax.xml.bind:jaxb-api:2.3.1 (*)
     +--- javax.xml.soap:javax.xml.soap-api:1.4.0
     \--- javax.jws:javax.jws-api:1.1
```

The previous release has a dependency to 
`com.sun.xml.ws:policy:2.7.2`. The new version contains some new transitive dependencies and the txw2 one seems to cause some issues. 

This is the error that Gradle presents:
```
* What went wrong:
Execution failed for task ':SSRKopiApi:genSources'.
> Could not resolve all files for configuration ':SSRKopiApi:jaxwsTools'.
   > Could not find org.glassfish.jaxb:txw2:2.4.0-b180608.0325.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/glassfish/jaxb/txw2/2.4.0-b180608.0325/txw2-2.4.0-b180608.0325.pom
       - https://repo.maven.apache.org/maven2/org/glassfish/jaxb/txw2/2.4.0-b180608.0325/txw2-2.4.0-b180608.0325.jar
     Required by:
         project :SSRKopiApi > com.sun.xml.ws:wscompile:2.3.1 > org.glassfish.jaxb:jaxb-jxc:2.3.1
         project :SSRKopiApi > com.sun.xml.ws:wscompile:2.3.1 > org.glassfish.jaxb:jaxb-jxc:2.3.1 > org.glassfish.jaxb:jaxb-runtime:2.3.1
   > Could not find org.glassfish.jaxb:txw2:2.4.0-b180608.0325.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/glassfish/jaxb/txw2/2.4.0-b180608.0325/txw2-2.4.0-b180608.0325.pom
       - https://repo.maven.apache.org/maven2/org/glassfish/jaxb/txw2/2.4.0-b180608.0325/txw2-2.4.0-b180608.0325.jar
     Required by:
         project :SSRKopiApi > com.sun.xml.ws:wscompile:2.3.1 > com.sun.xml.ws:policy:2.7.5
```

**Comment by giacgbj** [comment]
Same issue with org.jvnet.mimepull:mimepull:1.9.8:

![screenshot 2018-10-29 at 11 01 40](https://user-images.githubusercontent.com/1658549/47642900-33fc7700-db6a-11e8-8e08-5809db195098.png)

**Comment by westlake91361** [comment]
Same here:

[WARNING] The POM for com.sun.istack:istack-commons-runtime:jar:3.0.6 is missing, no dependency information available
[WARNING] The POM for com.sun.xml.fastinfoset:FastInfoset:jar:1.2.14 is missing, no dependency information available
[WARNING] The POM for org.glassfish.jaxb:txw2:jar:2.4.0-b180608.0325 is missing, no dependency information available
[WARNING] The POM for org.jvnet.mimepull:mimepull:jar:1.9.8 is missing, no dependency information available
[WARNING] The POM for commonj.sdo:commonj.sdo:jar:2.1.1.v201112051852 is missing, no dependency information available

**Comment by lukasj** [comment]
fixed in 2.3.2 in https://github.com/eclipse-ee4j/metro-jax-ws repo
